pkg/schemas/kops/Resource_KubeAPIServerConfig.generated.go

```diff
@@ -103,7 +103,8 @@ func ResourceKubeAPIServerConfig() *schema.Resource {
 			"etcd_quorum_read":                             OptionalBool(),
 			"request_timeout":                              OptionalDuration(),
 			"min_request_timeout":                          OptionalInt(),
-			"target_ram_mb":                                OptionalInt(),
```

Comment:
Have you tried an upgrade of an existing cluster? I'm wondering if we need a state migration to handle this being removed.

Reply:
Haven't tried. Not sure what happens to be honest.

Reply:
Running on one of my clusters now and it just started to update one of the control planes so should be fine I think. I don't have that configured though if that might make a difference.

Reply:
Yeah that's the more likely problem scenario. A cluster that is running that had previously set that value, and then tries to upgrade. When they attempt to upgrade the provider TF will probably give some error about an unknown value in state etc.

Reply:
I'll spin up a test-cluster with that configured and try to update it. Will report back when done.